recipes/libx265/all/conanfile.py

```diff
@@ -113,6 +113,8 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "x265"
```

Comment:
The problem here is that libx265's cmake script only creates and installs a pc files.
The policy of cci is to only set the names of the *official* installed cmake config scripts.

https://github.com/videolan/x265/blob/master/source/CMakeLists.txt

Reply:
I supposed that a cmake project name and pc file was enough to deduce cmake name, find scripts for libx265 or x265 are not widespread but it seems that x265 is a little more popular. If such a policy exists, no problem, I will drop it.

Reply:
I agree with @madebr, CCI recipes shouldn't suggest non official CMake names.

Reply:
Sounds reasonable. Dropped this chunk.